src/openrct2/title/TitleSequence.cpp

```diff
@@ -28,25 +28,20 @@
 
 #include <algorithm>
 #include <memory>
+#include <sstream>
 #include <vector>
 
 static std::vector<std::string> GetSaves(const utf8* path);
 static std::vector<std::string> GetSaves(IZipArchive* zip);
 static std::vector<TitleCommand> LegacyScriptRead(utf8* script, size_t scriptLength, const std::vector<std::string>& saves);
 static void LegacyScriptGetLine(OpenRCT2::IStream* stream, char* parts);
 static std::vector<uint8_t> ReadScriptFile(const utf8* path);
-static std::string LegacyScriptWrite(TitleSequence* seq);
 
-std::unique_ptr<TitleSequence> CreateTitleSequence()
-{
-    return std::make_unique<TitleSequence>();
-}
-
-std::unique_ptr<TitleSequence> LoadTitleSequence(const utf8* path)
+TitleSequence::TitleSequence(const utf8* path)
 {
     std::vector<uint8_t> script;
     std::vector<std::string> saves;
-    bool isZip;
+    bool is_zip = false;
```

Comment:
```suggestion
    bool isZip = false;
```
Use camelCase for local variables (won't conflict with the member variable because that is TitleCase)

(This is on commit https://github.com/OpenRCT2/OpenRCT2/pull/12815/commits/b0f7e8d558e04b4736b41257400ceab4cd73a38b)